app/helpers/application_helper.rb

```diff
@@ -80,17 +80,6 @@ def add_link_to_function(text, options)
         options.merge(:'data-original-title' => _("Click to add %s") % options[:"data-class-name"]))
   end
 
-  def add_html_classes(options, classes)
```

Comment:
This function was actually only used in one place and is way too complicated. removing it.